devblog/2023-04-28-offchain-data-transfer-postmortem.mdx

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Off Chain Data Transfer Post Mortem (4/18 & 4/19)
```

Comment:
It would be best to use a date format that is nation-agnostic, such as YYYY-MM-DD. (This goes for the whole document).

Reply:
changes made, thanks!